pure.zsh

```diff
@@ -222,6 +222,10 @@ prompt_pure_precmd() {
 		export VIRTUAL_ENV_DISABLE_PROMPT=12
 	fi
 
+	if [[ -n $IN_NIX_SHELL ]]; then
```

Comment:
It would be useful with a succinct code comment explaining what exactly this does and why.